File: LisApp/Models/ResultModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace LisApp.Models
{
    public class ResultModel
    {
        public long? IdResult { get; set; }

        public long IdEmployee { get; set; }

        public string EmployeeName { get; set; }

        public long IdStudy { get; set; }

        public DateTime DateOfResult { get; set; }

        public string Description { get; set; }

        public string ReasonForRepeat { get; set; }

        public bool Actual { get; set; }

        public List<ResultUnitModel> ResultUnitList { get; set; }

        public VerificationModel Verification { get; set; }

    }
}

File: LisApp/Scripts/Home/react/models/resultModel.js
import VerificationModel from './verificationModel';

class ResultModel {
    constructor()
    {
        this.DateOfResult = null;
        this.Description = null;
        this.ReasonForRepeat = null;
        this.EmployeeName = null;
        this.Verification = new VerificationModel();
    }
}

export default ResultModel;